packages/api-server/api_server/models/tortoise_models/tasks.py

```diff
@@ -29,6 +30,14 @@ class TaskState(Model):
     unix_millis_warn_time = DatetimeField(null=True, index=True)
     pickup = CharField(255, null=True, index=True)
     destination = CharField(255, null=True, index=True)
+    labels = ReverseRelation["TaskLabel"]
+
+
+class TaskLabel(Model):
+    state = ForeignKeyField("models.TaskState", null=True, related_name="labels")
+    label_name = CharField(255, null=False, index=True)
+    label_value_str = CharField(255, null=True, index=True)
+    label_value_num = BigIntField(null=True, index=True)
```

Comment:
Since the value come from json, technically we need both double and int. Or just double could be good enough if we don't work with values > max safe double (~48 bits iirc).

Reply:
https://github.com/open-rmf/rmf-web/pull/912/commits/7c2d8efa27ad3edacc438dec24e756bda3c2baa9